src/styles/atoms/_btn.scss

```diff
@@ -169,6 +169,24 @@
     width: 100%;
 }
 
+//  = FLAT BUTTON
+//  ---------------------------------------------------------------------------
+@mixin dc-btn--flat {
+    @include dc-link;
+
+    background: transparent;
+    border: 0;
```

Comment:
Would switch this for border-color:transparent. Border: 0 makes the flat button height 2px smaller than the other buttons.